app/configs/crashesListViewColumns.tsx

```diff
@@ -35,4 +45,165 @@ export const crashesListViewColumns: ColDataCardDef<CrashesListRow>[] = [
     label: "Collision",
     sortable: true,
   },
+  {
+    path: "vz_fatality_count",
+    label: "Fatalities",
+    sortable: true,
+  },
+  {
+    path: "sus_serious_injry_count",
+    label: "Sus serious injuries",
```

Comment:
idk why this shortening of suspected to "sus" bothers me but its what we currently have in prod and it keeps it from being so long so i guess its fine :shrug:

Reply:
yeah, "Sus" is not good. i removed it and fixed the casing (ty Chia). the [ACC designs](https://www.figma.com/proto/2acDnn8lRJ3FHnPEDGlk3A/Applied-Labs-Figma?type=design&node-id=811-60916&t=oEA7X290kfUxsKrT-0&scaling=scale-down-width&page-id=0%3A1&starting-point-node-id=811%3A60895&hotspot-hints=0&hide-ui=1) use the single-letter injury indicators which i could get on board with if we had a hover tooltip that explained what the letters meant